docs/release-notes/api-cli-access-token.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  API / CLI: Adds support for creating and administrating access tokens with user
+      defined lifespan intended for running automated workflows. This update provides users the
+      ability to securely generate and manage access tokens specifically designed for automation
+      while reducing operational friction and maintaining robust security protocols.
```

Comment:
- API/CLI: Add the ability to create and administer long-lived access tokens for use in automated workflows. Users can define the lifespan of these tokens, making it easier to securely authenticate and run processes without the need for frequent manual token generation. This feature enhances automation while maintaining strong security protocols by allowing tighter control over token usage and expiration.